.github/workflows/build_deploy.yaml

```diff
@@ -0,0 +1,148 @@
+name: Continuous Integration and Deployment of B2AI Peds Protocol
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set Environment Variables
+      run: |
+        echo "PROTOCOL_COMBINED_SLUG=peds-protocol-combined" >> $GITHUB_ENV
+        echo "PROTOCOL_QUESTIONNAIRES_SLUG=peds-protocol-questionnaires" >> $GITHUB_ENV
+    
+    - name: Echo Environment Variables
+      run: |
+        echo "PROTOCOL_COMBINED_SLUG is set to $PROTOCOL_COMBINED_SLUG"
+        echo "PROTOCOL_QUESTIONNAIRES_SLUG is set to $PROTOCOL_QUESTIONNAIRES_SLUG"
+    
+    - name: Check out repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0  # Fetches the entire history for comprehensive CI checks
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '21.x' # Updated to a more recent LTS version
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10' # Updated to Python 3.10
+
+    - name: Install system dependencies
+      run: sudo apt-get install jq
+
+    - name: Read Checksum from config.env
+      run: |
+        echo "Reading checksum..."
+        CHECKSUM=$(cat config.env | grep REPROSCHEMA_UI_CHECKSUM | cut -d '=' -f2)
```

Comment:
doing `source config.env` is a lot cleaner -> but i think config.env can be scrapped, see my below comment